package.json

```diff
@@ -24,36 +24,33 @@
   },
   "dependencies": {
     "@bugsnag/js": "6.5.2",
-    "@bugsnag/plugin-express": "6.5.1",
-    "@nestjs/common": "6.11.11",
-    "@nestjs/core": "6.11.11",
-    "@nestjs/microservices": "6.11.11",
-    "@nestjs/platform-express": "6.11.11",
-    "@nestjs/testing": "6.11.11",
-    "@nestjs/websockets": "6.11.11",
+    "@bugsnag/plugin-express": "6.4.1",
+    "@nestjs/common": "7.1.2",
+    "@nestjs/core": "7.1.2",
+    "@nestjs/platform-express": "7.1.2",
     "axios": "0.19.2",
     "body-parser": "1.19.0",
-    "chalk": "3.0.0",
+    "chalk": "4.0.0",
     "express-http-context": "1.2.3",
     "json-stringify-safe": "5.0.1",
-    "moment": "2.24.0",
+    "moment": "2.26.0",
     "reflect-metadata": "0.1.13",
-    "rxjs": "6.5.4",
-    "typescript": "3.8.3",
-    "uuid": "7.0.2"
+    "rxjs": "6.5.5",
+    "typescript": "3.7.2",
+    "uuid": "8.1.0"
```

Comment:
https://github.com/uuidjs/uuid/blob/master/CHANGELOG.md#800-2020-04-29

it is weird that we had it set already for v8 :-/ hmm :-/ but looks good